core/cap-0066.md

```diff
@@ -228,18 +224,23 @@ up to `txMaxReadEntries - 2` classic entries today (all classic entries + wasm +
 carefully measured assuming disk access only, so the addition of in-memory state should not negatively affect the network
 in any way should we maintain these values via disk read limits.
 
-TODO:  In-memory read entries must be measured in order to find a reasonable starting point.
+In-memory read limits are not required wrt execution time, as they are very inexpensive. However, the size of TX footprints
+does impact the cost and complexity of assembling transactions sets. For this reason, no ledger wide read limit is necessary,
+and no byte based read limit is necessary. However, a tx entry limit is still necessary to ensure efficient transaction set
+construction.
+
+TODO: Determine initial tx in-memory read limit.
```

Comment:
This value should be just inherited from the current value of `txMaxReadLedgerEntries` at the moment of network upgrade. It is the minimum viable value (or else we're risking to break someone).

Reply:
Sounds reasonable. I'll merge for now since we'll need to update this anyway with the new default values for write fees anyway with changing BucketList average to Soroban state size average